docs/StardustDocs/topics/installation.md

```diff
@@ -1,19 +1,19 @@
 [//]: # (title: Installation)
 
-You can use DataFrame in different environments — as any other JVM library.
-The following sections will show how to use DataFrame in [Jupyter](#jupyter-notebook), [Datalore](#datalore) and in a [Gradle project](#gradle).
+You can use Kotlin DataFrame library in different environments — as any other JVM library.
```

Comment:
Okay I like the clarification of "Kotlin DataFrame library", but it definitely needs a "the" everywhere you added it, haha. 
It's "the library", so also "the Kotlin DataFrame library". If that looks weird, then "Kotlin DataFrame" also works for me.